vtkParallelTransportFrame.h

```diff
@@ -26,10 +26,12 @@
 ///
 /// References:
 /// - Parallel transport theory: R. Bishop, "There is more than one way to frame a curve",
-///   American Mathematical Monthly, vol. 82, no. 3, pp. 246�251, 1975
+///   American Mathematical Monthly, vol. 82, no. 3, pp. 246�251, 1975
 /// - Parallel transport implementation: Piccinelli M, Veneziani A, Steinman DA, Remuzzi A, Antiga L.
 ///   "A framework for geometric analysis of vascular structures: application to cerebral aneurysms.",
 ///   IEEE Trans Med Imaging. 2009 Aug;28(8):1141-55. doi: 10.1109/TMI.2009.2021652.
+/// - Wang, W., J ̈uttler, B., Zheng, D., and Liu, Y. 2008. Computation of rotation minimizing frame. ACM Trans. Graph. 27, 1, Article 2 (March 2008), 18 pages.
```

Comment:
Please describe the most important characteristics of the two methods (especially when one should be used vs. the other), do a simple performance profiling (in release mode, compute directions for many large curves and see if the difference is noticeable).